apps/classroomio-com/src/routes/bootcamps/+page.svelte

```diff
@@ -0,0 +1,257 @@
+      <div class="w-fit py-2 px-4 rounded-full border-[#C2D2FF] border-2 text-center">
+        <p class=" text-sm md:text-base text-[#4B5563] text-center font-medium">Amazing benefits</p>
+      </div>
+
+      <div class="md:w-full flex flex-col items-start justify-start gap-y-4">
+        <h1 class="md:text-5xl text-3xl font-medium text-[#282828]">
+          Unlock Your Team's Potential and Drive Success
+        </h1>
+        <p class="text-[#656565] text-xl font-light">
+          Experience the power of collaborative learning with ClassroomIO. Engage your workforce
+          with personalized training that aligns with your company's goals.
+        </p>
+      </div>
+    </div>
+    <div class="w-full flex flex-col lg:w-[50%] lg:grid lg:grid-cols-2 gap-y-4 lg:gap-4">
+      <div
+        class="w-full md:w-[100%] md:p-6 p-4 border border-[#D9E0F5] rounded-2xl flex flex-col gap-y-6"
+      >
+        <div>
+          <img src="/employee-training/light-bulb.svg" alt="light-bulb" />
+        </div>
+        <div class="w-full flex flex-col justify-start align-start gap-y-2">
+          <h1 class="w-full text-xl font-medium font-[#282828]">Increased Engagement</h1>
+          <p class=" w-full text-base font-[#656565] font-light">
+            Interactive courses and collaborative features keep employees motivated.
+          </p>
+        </div>
+      </div>
+      <div class="w-full md:w-[100%] p-6 border border-[#D9E0F5] rounded-2xl flex flex-col gap-y-6">
+        <div>
+          <img src="/employee-training/learning.svg" alt="light-bulb" />
+        </div>
+        <div class="w-full flex flex-col justify-start align-start gap-y-2">
+          <h1 class="w-full text-xl font-medium font-[#282828]">Streamlined Learning</h1>
+          <p class=" w-full text-base font-[#656565] font-light">
+            Centralized platform for all training materials and progress tracking.
+          </p>
+        </div>
+      </div>
+      <div class="w-full md:w-[100%] p-6 border border-[#D9E0F5] rounded-2xl flex flex-col gap-y-6">
+        <div>
+          <img src="/employee-training/paint-brush.svg" alt="paint-brush" />
+        </div>
+        <div class="w-full flex flex-col justify-start align-start gap-y-2">
+          <h1 class="w-full text-xl font-medium font-[#282828]">Customizable Experience</h1>
+          <p class=" w-full text-base font-[#656565] font-light">
+            Tailor the learning experience to your company's specific needs and branding.
+          </p>
+        </div>
+      </div>
+      <div class="w-full md:w-[100%] p-6 border border-[#D9E0F5] rounded-2xl flex flex-col gap-y-6">
+        <div>
+          <img src="/employee-training/pie-chart.svg" alt="pie-chart" />
+        </div>
+        <div class="w-full flex flex-col justify-start align-start gap-y-2">
+          <h1 class="w-full text-xl font-medium font-[#282828]">Data-Driven Insights</h1>
+          <p class=" w-full text-base font-[#656565] font-light">
```

Comment:
also, the default font size is `text-base` so in most cases, you don't have to add the `text-base` attribute